webroot/js/snodas_map.js

```diff
@@ -230,14 +232,44 @@ query_selector.onclick = function(event) {
 var doy_html = 'Query Date<div id="snodas-doy-query"><div class="input-group input-daterange mb-3" id="snodas-doy-query-doy"><input type="text" class="input-sm form-control" id="snodas-doy-query-doy1" name="start"></div><select class="form-control" id="snodas-doy-query-years-start"></select>to<select class="form-control" id="snodas-doy-query-years-end"></select></div>';
 var variables_html = 'SNODAS Variable:<select class="form-control" id="snodas-query-variable"><option value="depth">Snow Depth</option><option value="swe" selected>Snow Water Equivalent</option><option value="runoff">Runoff</option><option value="sublimation">Sublimation</option><option value="sublimation_blowing">Sublimation (Blowing)</option><option value="precip_solid">Precipitation (Solid)</option><option value="precip_liquid">Precipitation (Liquid)</option><option value="average_temp">Average Temperature</option></select>';
 var regression = 'Forecast period:<select class="form-control" id="snodas-query-month-start"><option value="1">January</option><option value="2">February</option><option value="3">March</option><option value="4" selected>April</option><option value="5">May</option><option value="6">June</option><option value="7">July</option><option value="8">August</option><option value="9">September</option><option value="10">October</option><option value="11">November</option><option value="12">December</option></select>to<select class="form-control" id="snodas-query-month-end"><option value="1">January</option><option value="2">February</option><option value="3">March</option><option value="4">April</option><option value="5">May</option><option value="6">June</option><option value="7" selected>July</option><option value="8">August</option><option value="9">September</option><option value="10">October</option><option value="11">November</option><option value="12">December</option></select>';
 var submit = '<a url="https://api.snodas.geog.pdx.edu/" role="button" class="btn btn-success disabled" id="snodas-query-btn" aria-disabled="true">Submit Query</a>';
 
 function pp_table_init() {
+    var x = document.getElementById("awdb-select");
+    if (x.options.length == 0)
+    {
+        featuresPropertiesList.sort(function(x, y) {
+            const xPieces = x.awdb_id.split(":");
+            const yPieces = y.awdb_id.split(":");
+            var xTest = xPieces[1].concat(xPieces[0]);
+            var yTest = yPieces[1].concat(yPieces[0]);
+         if (xTest < yTest) {
+            return -1;
+         }
+         if (xTest > yTest) {
+            return 1;
+         }
+         return 0;
+      });
+      setOptions(featuresPropertiesList);
+      // Event listener to set the aoi on the map when it is selected from the list
+      x.addEventListener("change", selectMapLyr);
```

Comment:
I think you can remove this per the `select` element `onchange` property suggested above.